pom.xml

```diff
@@ -1446,12 +1446,13 @@
     <okioVersion>1.14.0</okioVersion>
     <opennmsApiVersion>0.6.0</opennmsApiVersion>
     <opennmsApiVersionOsgi>0.6.0</opennmsApiVersionOsgi>
-    <osgiVersion>6.0.0</osgiVersion>
-    <osgiCompendiumVersion>5.0.0</osgiCompendiumVersion>
-    <osgiEnterpriseVersion>5.0.0</osgiEnterpriseVersion>
+    <osgiVersion>7.0.0</osgiVersion>
+    <osgiAnnotationVersion>7.0.0</osgiAnnotationVersion>
+    <osgiCompendiumVersion>7.0.0</osgiCompendiumVersion>
+    <osgiEnterpriseVersion>7.0.0</osgiEnterpriseVersion>
     <owaspEncoderVersion>1.2.1</owaspEncoderVersion>
     <owaspHtmlSanitizerVersion>20170515.1</owaspHtmlSanitizerVersion>
-    <osgiJaxRsVersion>1.0.2.ONMS</osgiJaxRsVersion>
+    <osgiJaxRsVersion>1.0.3.ONMS-SNAPSHOT</osgiJaxRsVersion>
```

Comment:
Let's release this as `1.1.0.ONMS`, and point to stable release artifacts before merging.